src/main/java/com/redhat/digkins/services/CreateJobService.java

```diff
@@ -0,0 +1,40 @@
+package com.redhat.digkins.services;
```

Comment:
@wtrocki we should use org.aerogear namespace I think. Or, org.jboss.aerogear

An example: <https://github.com/aerogear/aerogear-android-authz/tree/master/aerogear-android-authz/src/main/java/org/jboss/aerogear/android/authorization/oauth2>

@matzew what do you say?